Makefile

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Missing a default action.

Reply:
I think the point is to either have an `all` or `help` rule here, that would either do everything we have to do or print some simple usage when you run `make` (which will use the first (default) rule, currently `terraform`).